topics/nativescript/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: nativescript, tns
+created_by: Progress
+display_name: NativeScript
+github_url: https://github.com/nativescript
+logo: nativescript.png
+released: March 2015
+short_description: NativeScript is a JavaScript-native mobile framework developed by Progress.
```

Comment:
I don't think we need `developed by Progress` in the short_desc; I don't feel strongly about this, though!